dlk/python/dlk/templates/include/global.tpl.h

```diff
@@ -35,15 +36,17 @@ template <typename pack_type>
 class QuantizedPacked {
  public:
   using T = pack_type;
+  using base_t = std::remove_cv_t<T>;
   static constexpr std::size_t BitCount = sizeof(pack_type) * CHAR_BIT;
   QuantizedPacked() = default;
   explicit QuantizedPacked(const T val) : val(val) {}
-  explicit operator T() const { return val; }
-  QuantizedPacked<T>& operator|=(const QuantizedPacked<T>& that) {
+  explicit operator base_t() const { return val; }
+  template <typename U, std::enable_if_t<std::is_same<std::remove_cv_t<T>, std::remove_cv_t<U>>::value, int> = 0>
```

Comment:
Maybe we can rewrite this as:
```c++
template <typename U, std::enable_if_t<std::is_same<base_t, std::remove_cv_t<U>>::value, int> = 0>
```
to make it easier to read?

Reply:
Fixed.